lib/__tests__/_createInsightsClient.test.ts

```diff
@@ -0,0 +1,32 @@
+import { createInsightsClient } from "../_createInsightsClient";
```

Comment:
Same comments as in https://github.com/algolia/search-insights.js/pull/390, feel free to apply or not.